src/abstractdatatable/join.jl

```diff
@@ -3,14 +3,17 @@
 ##
 
 # Like similar, but returns a nullable array
-similar_nullable{T}(dv::AbstractArray{T}, dims::Union{Int, Tuple{Vararg{Int}}}) =
```

Comment:
Please, this change is again totally unrelated to this PR. Let's keep is as simple as possible or the review will never end.